lib/StaticTensor.chpl

```diff
@@ -448,6 +448,18 @@ proc type staticTensor.matvecmul(m,v) {
     return matvec(m,v);
 }
 
+proc type staticTensor.nllLoss(
+    input: staticTensor(2,?eltType),
+    target: staticTensor(1,eltType),
+    weight: staticTensor(1, eltType) = ones(),
```

Comment:
`staticTensor.ones(eltType)` or make a function overload that doesn't have the `weight` argument and one that does have the `weight` argument, but with no default parameter.